### PR TITLE
fix artifacts not getting created

### PR DESCRIPTION
**Pull request by LegacyGamerHD**
this fixes my issue that my artifacts dont show in target folder

### CODE REVIEW

**Comment by LegacyGamerHD** [comment]
one check fails because its from my old pull to master it compiles fine

**Comment by LegacyGamerHD** [comment]
danggit it wasnt meant to be closed

File: bootstrap/bukkit/pom.xml [hunk 1]
```diff
@@ -48,6 +48,7 @@
                     </execution>
                 </executions>
                 <configuration>
+				    <finalName>${outputName}-BungeeCord</finalName>
```
Comment (RednedEpic) [inline bootstrap/bukkit/pom.xml]: The formatting is a bit off here.

**Comment by LegacyGamerHD** [comment]
doesnt work

**Pull request closed without merging**